ibm/service/kubernetes/data_source_ibm_container_vpc_cluster.go

```diff
@@ -255,7 +255,11 @@ func DataSourceIBMContainerVPCCluster() *schema.Resource {
 				Computed:    true,
 				Description: "email id of the key owner",
 			},
-
+			"image_security_enforcement": {
```

Comment:
please add this new parameter to the documentation

Reply:
fix `9c0a24e`